src/app/Routes.test.tsx

```diff
@@ -6,25 +6,115 @@ import configureStore from "redux-mock-store";
 
 import Routes from "./Routes";
 
-import baseURLs from "app/base/urls";
-import controllersURLs from "app/controllers/urls";
-import dashboardURLs from "app/dashboard/urls";
-import devicesURLs from "app/devices/urls";
-import domainsURLs from "app/domains/urls";
-import imagesURLs from "app/images/urls";
-import introURLs from "app/intro/urls";
-import kvmURLs from "app/kvm/urls";
-import machineURLs from "app/machines/urls";
-import poolsURLs from "app/pools/urls";
-import prefsURLs from "app/preferences/urls";
-import settingsURLs from "app/settings/urls";
-import subnetsURLs from "app/subnets/urls";
-import tagsURLs from "app/tags/urls";
-import zonesURLs from "app/zones/urls";
+import urls from "app/base/urls";
 import { rootState as rootStateFactory } from "testing/factories";
 
 const mockStore = configureStore();
 
+const componentPaths: { component: string; path: string }[] = [
```

Comment:
It doesn't look like this type is necessary (at least it doesn't give me any type errors if I remove it).

Reply:
It will fail if you use an incorrect value for the URL (e.g. object instead of a string). That's what happened to me when refactoring, the test was failing without a clear indication of why.